apps/backend/app/api/routes/transformations.py

```
@@ -1511,4 +1522,372 @@ async def clear_history(
+        raise
+    except Exception as e:
+        logger.exception(f"Error cancelling bulk job: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error") from e
+
+
+@router.post(
+    "/datasets/{dataset_id}/bulk-jobs/{job_id}/retry",
+    response_model=BulkTransformationJobResponse
+)
+async def retry_bulk_job(
+    dataset_id: str,
```

Ruff (0.14.10)

Unused function argument: `dataset_id`

(ARG001)